Sources/BraintreeCore/ConfigurationLoader.swift

```diff
@@ -39,26 +41,32 @@ class ConfigurationLoader {
             completion(cachedConfig, nil)
```

Comment:
nit pick : this is where I meant to put the comment about putting this on main since get closure is called from main by default in our code base, I think.

Reply:
I'm a little confused what this means - can you clarify?

Reply:
https://github.com/braintree/braintree_ios/blob/b4ff6ce555b099b57c2ef74e225998fd262164b1/Sources/BraintreeCore/BTHTTP.swift#L279
This is defaulted to main and I haven't seen anywhere where we override it. So I was thinking if merchant's app expects this and other completions to be returned on main.

Reply:
Hm, I don't think there's anything additional we need to be doing at the Config level, since this level is essentially just propagating up whatever result we get back from the BTHTTP layer. If the BTHTTP layer is switching over to the main thread to deliver it's result, then the ConfigLoader is also delivering on the main thread once the result is received.

I don't think we should change any thread management logic in this PR.